src/omv/sensor.c

```diff
@@ -1017,6 +1100,11 @@ void DCMI_DMAConvCpltUser(uint32_t addr)
             // that will cause data loss if we make the DMA hardware write directly to the FB.
             //
             uint16_t size = __REV16(*src16);
+            // Prevent a buffer overflow when writing the jpeg data.
+            if (line + size > OMV_RAW_BUF_SIZE) {
```

Comment:
Just a note, this should be called something else, offset maybe, it's not just a line count anymore.

Reply:
I'll make a union.

Reply:
Renamed to offset.